src/applications/burials/components/IntroductionPage.jsx

```diff
@@ -136,7 +117,6 @@ class IntroductionPage extends React.Component {
         </div>
         <SaveInProgressIntro
           buttonOnly
-          prefillEnabled={this.props.route.formConfig.prefillEnabled}
```

Comment:
Should this stay?

Reply:
Good catch, I think I accidentally removed this